apps/dialtone-documentation/docs/components/stack.md

```diff
@@ -156,6 +156,45 @@ vueCode='
 '
 showHtmlWarning />
 
+Set gap to 600 at extra large screen size, 500 for large screen size, 400 for medium screen size, and 300 for small screen size
```

Comment:
If you were reading just this page you would not know what a large screen size, medium screen size etc is. We should probably link to our current responsive page to show the sizes (I know it's kinda wrong right now due to the reversal issue).

https://dialtone.dialpad.com/utilities/responsive/breakpoints.html